rendering_styles/topo.render.xml

```diff
@@ -2032,7 +2032,14 @@
 					<case minzoom="15" textSize="12" tag="natural" value="saddle" nameTag="" nameTag2="ele"/>
 					<case minzoom="13" textSize="12" tag="natural" value="gorge" textOnPath="true" textOrder="44" textDy="0"/>
 					<case minzoom="13" textSize="12" tag="natural" value="couloir" textOnPath="true" textOrder="44" textDy="0"/>
-					<case minzoom="10" textSize="13" tag="region:type" value="mountain_area" textOrder="43"/>
+					<switch tag="region:type" value="mountain_area" textHaloRadius="5" textOrder="1" textDy="0">
```

Comment:
Same here. Topo is topographic style but placing mountain areas over all other text labels is arguable.

Reply:
Same as before. I'll do again all tests with *textOrder="33"*